cosmos/runtime/init.sh

```diff
@@ -84,25 +89,6 @@ if [[ $overwrite == "y" || $overwrite == "Y" ]]; then
 	jq '.app_state["mint"]["params"]["mint_denom"]="abera"' "$GENESIS" >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
```

Comment:
You can delete all this no?

Reply:
failed to do so. genesis.json has `genesis_time` and account addresses generated